Cargo.toml

```diff
@@ -16,6 +16,7 @@ Windows, macOS, and Linux, as well as WebAssembly via Emscripten.
 """
 keywords = ["artichoke", "artichoke-ruby", "mri", "cruby", "ruby"]
 categories = ["command-line-utilities"]
+rust-version = "1.57.0"
```

Comment:
Sorry to nitpick here, but I consider the `rust-version` field semantically linked to `edition`. Would you mind moving this line to line 7 directly below the `edition = "2021"` line?

Can we make this change for all of the `Cargo.toml`s?